t/apicast-policy-keycloak-role-check.t

```diff
@@ -393,3 +393,160 @@ yay, api backend
 --- no_error_log
 [error]
 oauth failed with
+
+
+=== TEST6: Role check with allow methods
```

Comment:
Can you add a space between `TEST` and `6` ? It's just because I don't know if that can cause some issue with the script that we have to automatically number tests correctly? https://github.com/3scale/APIcast/blob/master/script/reorder-tests

Reply:
I've updated all the test.